### DISCUSSION TITLE
Single or multiple tooltip components within project?

### DISCUSSION THREAD

**Discussion by brandstetterm** [ReactTooltip/react-tooltip#1177] [Q&A] [answered]
Hello, I am currently wondering what the best procedure is for using react-tooltip. Is it better to have a single global tooltip component with an id, which is then used by all components with the `data-tooltip-id` data attribute? Or is it better to have multiple tooltip components, for example for every place of use?

**Comment by gabrieljablonski** [comment, marked as answer]
It's usually a good idea to start off with just a single global tooltip, and then adding extra ones if the need ever arises.

Examples of situations where you'll probably want extra tooltips so it's easier to work with them:

- Different styling (background color, padding, ...)
- Different behavior (opening on click, show/hide delay)

There are probably other situations as well, but hopefully you get the idea.

**Reply by brandstetterm** [reply]
Thank you very much for your quick answer :) 
Since we have a consistent design accross our tooltips, it is probably best to use a single global one
